helper/schema/schema.go

```diff
@@ -91,8 +91,38 @@ type Schema struct {
 	// If CustomizeDiffFunc makes this field ForceNew=true, the
 	// following DiffSuppressFunc will come in with the value of old being
 	// empty, as if creating a new resource.
+	//
+	// By default, DiffSuppressFunc is considered only when deciding whether
+	// a configuration value is significantly different than the prior state
+	// value during planning. Set DiffSuppressAlways to make this rule take
+	// effect more broadly, so ResourceData.Set will ignore all attempts to
+	// set a value that's non-equal but equivalent.
 	DiffSuppressFunc SchemaDiffSuppressFunc
 
+	// DiffSuppressOnRefresh enables using the DiffSuppressFunc to undo
+	// normalization-classified changes during a "Refresh" call, in addition
+	// to the default behavior of doing so during "Diff".
```

Comment:
Two potential items to consider here:

- While the "Refresh" and "Diff" terms may be familiar for SDK and core developers, I'm not sure provider developers will understand the same terminology. I'd suggest referring back to the SDK's provider-facing terminologies/abstractions in this case, such as mentioning calls to ResourceData.Set in provider logic.
- I'd be hesitant to mention "undo", while that may be an implementation detail, I think provider developers can think of this as "ignoring" the change.

I think the second half of the last paragraph may actually be good to include up here, for upfront context to provider developers.